apps/fluent-tester/src/TestComponents/Icon/IconTest.tsx

```diff
@@ -78,6 +78,7 @@ const Icons: React.FunctionComponent = () => {
             shouldShowDataUri ? <Icon svgSource={svgD20DataUriProps} width={100} height={100} color="#7a7" /> : null
           }
           <Icon svgSource={svgUriProps} width={100} height={100} color="red" />
+          <Icon svgSource={svgProps} width={100} height={100} color={PlatformColor('WindowText')} accessibilityLabel="Wheelchair" />
```

Comment:
Can we make sure this is an platform.os check? It'll crash on other platforms I think

Reply:
Oh yep, sorry about that

Reply:
It would be nice to pull this out and test multiple platforms' platform colors, but we can do that later!